packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -738,7 +738,7 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
                 if (supportCut) {
                     document.execCommand('cut');
                 } else {
-                    this.messageService.warn("Please use the browser's cut command or shortcut.");
+                    this.messageService.warn(nls.localize('theia/workspace/cut', "Please use the browser's cut command or shortcut."));
```

Comment:
Suggestion: Why use `theia/workspace/...` as the key? We are in the `@theia/core` package. Also, the key should explain a bit more to prevent accidental ambiguities in the future.
```suggestion
                    this.messageService.warn(nls.localize('theia/core/cutWarn', "Please use the browser's cut command or shortcut."));
```

Reply:
Okey, and we have one more key, it had been before my changes. Should I correct it, or not?

https://github.com/eclipse-theia/theia/blob/4cb425c9012f72d3101b482252f9b21322860fb2/packages/core/src/browser/common-frontend-contribution.ts#L934